specs/interop/predeploys.md

```diff
@@ -86,6 +90,142 @@ To ensure safety of the protocol, the [Message Invariants](./messaging.md#messag
 
 [`Identifier`]: ./messaging.md#message-identifier
 
+### Access-list
+
+Execution of messages is statically pre-declared in transactions,
+to ensure the cross-chain validity can be verified outside the single-chain EVM environment constraints.
+
+After pre-verification of the access-list, the `CrossL2Inbox` can allow messages
+to execute when there is a matching pre-verified access-list entry.
+
+Each executing message is declared with 3 typed access-list entries:
+- 1: Lookup identity
+- 2: ChainID extension
+- 3: Checksum
```

Comment:
Since we last discussed the entry formatting, I realized it's good to have typing for each of them, and that the chain-ID can really just be an extension entry, and we don't need it by default if it already fits in the lookup identity entry.